### DISCUSSION TITLE
Ability to create folders to sort your favourite beatmap tab on the profile page in osuweb

### DISCUSSION THREAD

**Discussion by MelZeee** [ppy/osu#19011] [Ideas] [answered]
I would like to request a feature to sort out the favourite tab on the user page in osuweb with folders that you can create and name. And when you want to favourite a new beatmap you will be prompted with a option to add that map in that specific folder. This will make it more cleaner when previewing favourite beatmaps and it will also be easier to find a specific style of beatmap, i know we have "collections" in the client but would also be nice seeing something similar on the profile page

**Comment by smoogipoo** [comment, marked as answer]
Should be suggested at https://github.com/ppy/osu-web first.

**Reply by MelZeee** [reply]
Thank you for letting me know i will go over and post the suggestion in osu-web
edit: https://github.com/ppy/osu-web/issues/9090
